src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -14,69 +10,39 @@ export function WalletDropdownFundLink({
   rel,
   target,
   text = 'Fund wallet',
+  fundingUrl,
```

Comment:
since all these defaults were moved to WalletDropdownFundLinkButton, do we need them here?  could { fundingUrl, ...rest }.  This is less explicit, but imo nice as it shows this component only acts on fundingUrl.  @0xAlec better than the destructure spread?

Reply:
perhaps, personally I don't consider prop spreading an anti-pattern, especially with typescript.

Reply:
@alessey I feel the same, but every other component in OnchainKit avoid doing this and I think consistency is very important. Probably something to align on as a team rather than in a PR, and then either add the no props spread eslint rule or don't and say its ok.

Reply:
@alessey ok updated the props!